packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewYoutube.html

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="iframe-wrapper iframe-wrapper__youtube">
+    <div
+      onclick={stopPropagation}
+      class="quickview__spinner-container slds-is-relative wrapper"
+    >
+      <lightning-spinner
+        if:true={isLoading}
+        alternative-text="Loading"
+        size="large"
+      ></lightning-spinner>
+      <iframe
+        id="quickview__content-iframe"
+        class="quickview__content-container slds-modal__content slds-p-around_large slds-wrap"
+        src={youtubeURL}
+        height="100%"
+        width="100%"
+        onload={handleIframeLoaded}
+        allowfullscreen
+        referrerpolicy="same-origin"
+        sandbox="allow-same-origin allow-scripts allow-popups allow-top-navigation allow-top-navigation-by-user-activation allow-presentation"
```

Comment:
Do we need all this directives to make it work?
Let's keep only what we need.

Reply:
I removed the sandbox, it is not needed. However, I originally put the permissions because I was getting errors in the console. I was able to avoid console errors after removing the sandbox by allowing youtube as a trusted site in the CSP Trusted Sites settings of the scratch org.

https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/csp_trusted_sites.htm